core/templates/dev/head/domain/utilities/BrowserCheckerService.js

```diff
@@ -24,6 +24,15 @@ oppia.factory('BrowserCheckerService', function() {
   var isIEedge = winNav.userAgent.indexOf('Edge') > -1;
   var isIOSChrome = winNav.userAgent.match('CriOS');
 
+  // For details on the reliability of this check, see
+  //https://stackoverflow.com/questions/9847580/
```

Comment:
Add space after //, here and on the next line.

Reply:
Done!